test/cdk-test-integration/lib/cdk-test-integration-stack.ts

```diff
@@ -1,16 +1,55 @@
+//import * as path from 'path';
 import { Stack, StackProps } from 'aws-cdk-lib';
 import * as ddb from 'aws-cdk-lib/aws-dynamodb';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
 import { Construct } from 'constructs';
 import { GlobalTable } from '../../../src';
 
 export class CdkTestIntegrationStack extends Stack {
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
-    new GlobalTable(this, 'Table', {
+    const my_table = new GlobalTable(this, 'Table', {
       partitionKey: {
         name: 'id',
         type: ddb.AttributeType.STRING,
       },
+      replicas: [{
+        region: 'us-east-1',
+      },
+      {
+        region: 'us-west-1',
+      },
+      {
+        region: 'us-west-2',
+      }],
+      tableName: 'with_replica_table_name',
```

Comment:
With a hard-coded physical name, is your integration test succeeding after the first time? Currently, only a retention policy of RETAIN is possible. So any deployment of this stack after the first deployment+deletion to the same environment will fail. It looks like it's not failing right now, because the tableName property is not actually be honored as the physical name.